testlog.txt

```diff
@@ -0,0 +1,528 @@
+/Users/barnhark/projects/landlab/landlab/landlab/utils/decorators.py:134: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() or inspect.getfullargspec()
```

Comment:
@Giuseppecipolla95 can you delete this file? I must have accidentally added it. My apologies. I didn't mean to. It was used for debugging purposes only.